dc_utils/templates/dc_base.html

```diff
@@ -45,7 +46,54 @@
             {% endblock main_base %}
 
             {% block footer_base %}
+                <footer class="ds-footer">
+                    <div class="ds-block-centered ds-text-centered ds-stack">
+                        {% block mailing_list %}
+                            <div class="ds-dark">
+                                <a class="ds-cta ds-cta-blue" href="https://mailinglist.democracyclub.org.uk/subscription/form">
+                                    Join our mailing list
+                                </a>
+                            </div>
+                        {% endblock mailing_list %}
+                        {% block footer_menu %}
```

Comment:
I think in the base template this block should exist but be empty. We don't want to risk this content coming through on other projects. We can overwrite this block in the demo project template